containers/covabot/src/cova-bot/covaBot.ts

```
@@ -0,0 +1,307 @@
+					if (!identity) {
+						logger.debug(`[CovaBot] Identity resolution failed for trigger "${trigger.name}" - bot will remain silent`);
+						continue; // Skip this trigger, bot remains silent
+					}
+
+					// Send message with Cova's identity
+					await this.sendMessage(message, responseText, identity);
+					return; // Exit after first successful response
+
+				} catch (error) {
+					logger.error(`[CovaBot] Error in trigger "${trigger.name}":`, error as Error);
+					continue; // Try next trigger
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)